src/coffee/directives/api/map.coffee

```diff
@@ -243,7 +243,7 @@ angular.module('uiGmapgoogle-maps.directives.api')
 
             scope.$watch 'bounds', (newValue, oldValue) ->
               return  if newValue is oldValue
-              if !newValue.northeast.latitude? or !newValue.northeast.longitude? or !newValue.southwest.latitude? or !newValue.southwest.longitude?
+              if !newValue?.northeast?.latitude? or !newValue?.northeast?.longitude? or !newValue?.southwest?.latitude? or !newValue?.southwest?.longitude?
```

Comment:
Sometimes on app load I would get a js error stemming from this line, where apparently `newValue.northeast` was undefined, and that would result in the map completely locking down and not responding to any user actions.

I'm not sure if this will fix the temporary locks I was seeing.

@nmccready